db/migrate/20190805140757_create_image_table.rb

```diff
@@ -13,12 +14,11 @@ def up
 
     Icon.all.each do |icon|
       image = Image.create!(
-        :content   => icon.data,
-        :extension => "svg",
+        :content   => Base64.strict_encode64(icon.data),
         :tenant_id => icon.tenant_id
       )
 
-      icon.update(:image_id => image.id)
+      icon.image = image
```

Comment:
Does this work? There's no save here.

Reply:
There is a test to validate that it does :) 
https://github.com/lindgrenj6/catalog-api/blob/76bb631a425d69928a7684c8dadd95bdd248aeb6/spec/services/catalog/update_icon_spec.rb#L17-L22

Reply:
You're not calling `Catalog::UpdateIcon` here, you're just doing `icon.image = image`, no?

Reply:
Yep you're right, I derped hard because Friday. Updated The migration to use `icon.create_image!` instead